core/ui/src/views/ClusterStatus.vue

```diff
@@ -144,13 +147,38 @@
           class="min-height-card"
         >
           <template slot="content">
-            <NsButton
-              kind="ghost"
-              :icon="ArrowRight20"
-              @click="$router.push('/nodes')"
-            >
-              {{ $t("cluster_status.go_to_nodes") }}
-            </NsButton>
+            <div class="card-rows">
+              <div class="card-row">
+                <div v-if="!loading.getClusterStatus">
+                  <div
+                    v-if="nodesOffline.length"
+                    class="card-row icon-and-text"
+                  >
+                    <NsSvg :svg="ErrorFilled16" class="icon ns-error" />
+                    <span>
+                      {{
+                        $tc("nodes.nodes_offline_c", nodesOffline.length, {
+                          num: nodesOffline.length,
+                        })
+                      }}
+                    </span>
+                  </div>
+                  <div v-else class="card-row icon-and-text">
```

Comment:
Maybe it's better to show "All nodes are online" only if the cluster has more than one node